src/match_stream_client.erl

```diff
@@ -80,6 +81,19 @@ wait_for_socket(Other, State) ->
 wait_for_params({data, Payload}, State = #state{peerport = PeerPort}) ->
   ?DEBUG("Data received from client on ~p:~p~n",[PeerPort, Payload]),
   case binary:split(Payload, [<<":">>, <<$\r>>, <<$\n>>], [global]) of
+    [<<"V">>, <<"2">>, Uid | _] ->
+      try
+        case match_stream:matches() of
+          [MatchId | _] ->
+            ok = match_stream_user:watch(Uid, MatchId, self());
+          [] ->
+            ok = tcp_send(State#state.socket, frame(io_lib:format("ERROR: No matches to watch~n", [])), State)
```

Comment:
Line 90 is too long:             ok = tcp_send(State#state.socket, frame(io_lib:format("ERROR: No matches to watch~n", [])), State).